packages/notebook/src/widget.ts

```diff
@@ -2672,10 +2672,14 @@ export class Notebook extends StaticNotebook {
       // We don't want to prevent the default selection behavior
       // if there is currently text selected in an output.
       const hasSelection = (window.getSelection() ?? '').toString() !== '';
+      // We don't want to block the shift-click mouse up handler
+      // when the current cell is the active cell.
+      const isActiveCell = index == this.activeCellIndex;
       if (
         button === 0 &&
         shiftKey &&
         !hasSelection &&
+        !isActiveCell &&
```

Comment:
Giving it a bit more thought I think this has to be more subtle. The idea is that user can press Shift, press mouse down and start dragging cursor to select cells. Instead of short-circuiting here I think we should store the `activeCellIndex` if starts shift + mousedown on active cell and skip preventing default on `_evtDocumentMouseup()` if the cell index has not changed.

Reply:
I will push a commit